statsmodels/tsa/tests/test_stattools.py

```diff
@@ -990,6 +991,30 @@ def test_acovf_fft_vs_convolution(demean, adjusted):
     assert_almost_equal(F1, F2, decimal=7)
 
 
+@pytest.mark.parametrize("demean", [True, False])
+@pytest.mark.parametrize("adjusted", [True, False])
+def test_ccovf_fft_vs_convolution(demean, adjusted):
+    np.random.seed(1)
+    x = np.random.normal(size=128)
+    y = np.random.normal(size=128)
+
+    F1 = ccovf(x, y, demean=demean, adjusted=adjusted, fft=False)
+    F2 = ccovf(x, y, demean=demean, adjusted=adjusted, fft=True)
+    assert_almost_equal(F1, F2, decimal=7)
+
+
+@pytest.mark.parametrize("demean", [True, False])
+@pytest.mark.parametrize("adjusted", [True, False])
+@pytest.mark.parametrize("fft", [True, False])
+def test_compare_acovf_vs_ccovf(demean, adjusted, fft):
```

Comment:
pass `reset_randomstate` as an argument to the test function. Then you do not need to set `np.random.seed` internally.

https://github.com/statsmodels/statsmodels/blob/77bb1d276c7d11bc8657497b4307aa7575c3e65c/statsmodels/stats/tests/test_statstools.py#L86

This is a global fixture.